### PR TITLE
Update es.js

### PR DESCRIPTION
**Pull request by mocherz**
mostly changed "backup" translation to "archivo"

### CODE REVIEW

File: es.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ export const es = {
     history: "Historia",
     price: "Precio",
     usd: "USD",
-    txs: "TX",
+    txs: "impuestos",
```
Comment (alexanvl) [inline es.js]: This is an abbreviation for transactions

File: es.js [hunk 2]
```diff
@@ -194,38 +194,38 @@ export const es = {
     screen_not_exist: "Esta pantalla no existe.",
     go_to_home_screen: "Ir a la pantalla de inicio",
     gift_cards: "Tarjetas de regalo",
-    backup_wallet: "Hacer una copia de seguridad de tu nueva billetera",
+    backup_wallet: "Hacer un archivo de seguridad de tu nueva billetera",
     backup_required:
-      "Se requiere una copia de seguridad para proteger tus fondos",
-    why_backups_important: "¿Por qué son importantes las copias de seguridad?",
+      "Se requiere un archivo de seguridad para proteger tus fondos",
+    why_backups_important: "¿Por qué son importantes l0s archivos de seguridad?",
```
Comment (alexanvl) [inline es.js]: Looks like a typo "0" -> "o"

File: es.js [hunk 3]
```diff
@@ -245,7 +245,7 @@ export const es = {
     choose_dogeverse_username: "Elige una etiqueta DogeTag",
     dogeverse_link: "Tu enlace web será",
     continue: "Continuar",
-    dogeverse_entry_title: "Has entrado en el verso",
+    dogeverse_entry_title: "Has entrado en el dogeverse",
```
Comment (alexanvl) [inline es.js]: Please capitalize Dogeverse

**Review by alexanvl — CHANGES_REQUESTED** [review]
Please see the the suggested changes, thanks!

File: es.js (hunk 1 above)
Comment (mocherz) [reply es.js]: done

File: es.js (hunk 2 above)
Comment (mocherz) [reply es.js]: sorry, fixed...

File: es.js (hunk 3 above)
Comment (mocherz) [reply es.js]: done

**Review by alexanvl — APPROVED** [review]

**Pull request merged**